crates/model/src/solver_competition.rs

```diff
@@ -130,6 +144,11 @@ pub enum Score {
     #[serde(rename = "scoreProtocol")]
     #[serde(with = "u256_decimal")]
     Protocol(U256),
+    /// The score is calculated by the protocol and success_probability provided
+    /// by solver is taken into account
+    #[serde(rename = "scoreProtocolRisk")]
+    #[serde(with = "u256_decimal")]
+    ProtocolWithRisk(U256),
```

Comment:
sorry for being so nit-picky but can we call it something like: ProtocolWithSolverRisk

Reply:
done